### PR TITLE
Language Independence

### PR DESCRIPTION
**Pull request by stevehoag**
Ported from MSDN. Eliminated last section on Cross-Language Interoperability since we only support C# at present. This section should be rewritten for .NET Core and added back in once we support multiple languages.

### CODE REVIEW

**Comment by svick** [comment]
I think `TypedReference` is not supported on .Net Core (which is why it doesn't have an API reference link). Why is it even mentioned?

**Comment by svick** [comment]
```md
the getter is named **get**_*propertyname* and the setter is **set**_*propertyname*
```

GitHub-flavored Markdown understands the two underscores as italics:

the getter is named **get**_*propertyname* and the setter is **set**_*propertyname*

This can be fixed by escaping them with `\`:

```md
the getter is named **get**\_*propertyname* and the setter is **set**\_*propertyname*
```

Renders correctly as:

the getter is named **get**\_*propertyname* and the setter is **set**\_*propertyname*

**Comment by richlander** [comment]
I believe that F# now works. Does VB not work?

**Comment by stevehoag** [comment]
@richlander My understanding was that we're only supporting C# for RC2. I assumed that was because we aren't testing F# or VB, presumably because of a lack of resources. I also understood that VB support won't be coming until after RTW - not sure who told us that. Because of that assumption we've been removing the VB content from the .NET Framework content that we're porting, planning on adding it once VB is fully supported.

So, what is the official story for F# and VB on ,NET Core?

**Comment by richlander** [comment]
It looks like we're overlapping scope between "essentials" and "concepts". 

My proposal: Move all the CLR concepts and keep the BCL content in essentials.

Then, for this doc, make it general, not .NET Core specific (so F#, C# and VB).

**Comment by mairaw** [comment]
Rename this file to lowercase too.

File: docs/essentials/languageIndependence.md [hunk 1]
```diff
@@ -1 +1,1653 @@
...
+    }
+    ```
+
+### Interfaces
+
+CLS-compliant interfaces can define properties, events, and virtual methods (methods with no implementation). A CLS-compliant interface cannot have any of the following: 
+
+* Static methods or static fields. The C# compiler generatse compiler errors if you define a static member in an interface. 
+
+* Fields. The C# acompiler generates compiler errors if you define a field in an interface.
```
Comment (richlander) [inline docs/essentials/languageIndependence.md]: "acompiler"

File: docs/essentials/languageIndependence.md [hunk 2]
```diff
@@ -1 +1,1653 @@
...
+         ti.Recorded = DateTimeOffset.UtcNow;
+         previous = current;
+         current = value;
+         if (Math.Abs(current - previous) >= tolerance) 
+            raise_TemperatureChanged(new TemperatureChangedEventArgs(previous, current, ti.Recorded));
+      }
+   }
+
+   public void raise_TemperatureChanged(TemperatureChangedEventArgs eventArgs)
+   {
```
Comment (richlander) [inline docs/essentials/languageIndependence.md]: "In.NET"

**Comment by richlander** [comment]
This is a long topic. I'm reflecting on it. Who do you think the audience is for it? Put another way, in what use case does one need to know CLS compliance anymore?

Put another way, this is not "essentials" content. it's more "concepts", but it's also something that <1% of our users need to know. Should we just leave this to readers of the ECMA spec?

**Comment by stevehoag** [comment]
Closing this PR - will resubmit once content is moved to concepts.

**Pull request closed without merging**